test/common/lock_free_array_test.cpp

```diff
@@ -99,5 +95,96 @@ TEST_F(LockFreeArrayTests, SharedPointerTest2) {
   }
 }
 
+TEST_F(LockFreeArrayTests, FindValidAndEraseTest) {
+  typedef uint32_t value_type;
+
+  {
+    LockFreeArray<value_type> array;
+
+    value_type invalid_value = 6288;
```

Comment:
Random number?

Reply:
Just any sentinel value for this test that wasn't inserted will do.

Reply:
Please use `INVALID_OID` instead.